### ISSUE TITLE
Enforce consistent data formats (HTX HXL tagging for CSV)

### ISSUE THREAD

**Issue by ghost** [thinkingmachines/unicef-ai4d-research-bank#39]

**Comment by AnthonyMockler** [comment]
https://data.humdata.org/tools/hxl-example/

**Comment by AnthonyMockler** [comment]
Can also do this for JSON formatted
https://hxlstandard.org/standard/1-1final/tagging/#tagging.json

**Comment by butchtm** [comment]
Hi @AnthonyMockler  we might need to revisit this as a validation condition as this will prevent the current poverty mapping and air quality datasets as they don't currently implement HXL tags. 

As an alternative, we can probably add a warning instead and a message to encourage the data provider  to apply hxl tags to the datasets they are contributing.

**Comment by AnthonyMockler** [comment]
@butchtm Yep happy to talk - Is there a link to the output csvs of the current air quality / poverty metrics? 

(I still think it's a good criterion, and we should explore  / estimate what it would take to automate 'fixing' of incoming data that doesn't already have HXL tags)

**Comment by butchtm** [comment]
linking the[ air quality repo](https://github.com/thinkingmachines/unicef-ai4d-air-quality) and [output csvs (on google drive)](https://drive.google.com/drive/folders/1c4f1TJzW7uI9IgqXZY_08pJb1YvpevG1):

**Comment by butchtm** [comment]
Hi @AnthonyMockler 
After further review of the [hxl tagging json recommendations](https://hxlstandard.org/standard/1-1final/tagging/#tagging.json), it seems that the [geojson format](https://www.rfc-editor.org/rfc/rfc7946) is not currently compatible with the guidelines. There was some discussion in the [hxlstandard google groups way back in 2018](https://groups.google.com/g/hxlproject/c/N-606LwRz80/m/Q7dq88YzBAAJ) but no further developments have been announced. 

So the final recommendation is not to require geojson files to have HXL tags, since there is no current way to add them without invalidating the geojson format.
